### ISSUE TITLE
ABR letters layout

### ISSUE THREAD

**Issue by nathankota** [uic-utah/uic-inventory#282]
ABR letters are generated according to the inventory type:
  - [General](https://docs.google.com/document/d/1W4Sm_-vEFRhq2Z_-uE2mw_gtmlK6-TaZw1SCB4PrxQ4/)
  - [LUWDS](https://docs.google.com/document/d/11gcwbUdaEWWpzI_n7UfZcKrVzUtHfJEU3u3xe5a_nO4/)
  - [Storm water drainage](https://docs.google.com/document/d/1PH6eNWjaJ0YTirWOC56P1yQftyDxtresdUCSxjVWTF8/)
  - [SER](https://docs.google.com/document/d/1-jptVL4pIhDmcWN26Z4TdusHNR_tVS5ntjIz9qoFHLo/)
  - [Vet/kennel/grooming](https://docs.google.com/document/d/1Sw6S3tl0OO5N2WMks0UJAJgof1Iz6DfFzs-ohLHYebA/)

Once an inventory is approved, the `Download` button on the Inventory Review page will generate the appropriate ABR letter and collate it with the inventory report (#273) into a [single pdf document](https://docs.google.com/document/d/16-ri5ux1-8d--hUtMINO_oWBVt5z45CqjugJfoMwCT4/).

**Comment by steveoh** [comment]
How do these look?

[arb-general.pdf](https://github.com/uic-utah/uic-inventory/files/13746866/arb-general.pdf)
[arb-luwds.pdf](https://github.com/uic-utah/uic-inventory/files/13746867/arb-luwds.pdf)
[arb-ser.pdf](https://github.com/uic-utah/uic-inventory/files/13746868/arb-ser.pdf)
[arb-vet.pdf](https://github.com/uic-utah/uic-inventory/files/13746869/arb-vet.pdf)
[arb-water.pdf](https://github.com/uic-utah/uic-inventory/files/13746870/arb-water.pdf)

**Comment by rsparker-utah** [comment]
The ABR letter templates are looking really good! I just had a few minor comments from my review:

1.) This comment applies to all of the ABR letter templates. On the second page, under the “PROHIBITION OF UNAUTHORIZED INJECTION” section, could we replace the second paragraph starting with “If any of the injection wells…”:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/0952e1f0-47cf-43fa-9bb6-fd1af43d6cb9)

with the following paragraph:

If at any time the Director of the Utah Division of Water Quality determines that a Class V well may cause a violation of primary drinking water rules, the Utah UIC Program Rules require the Director to take appropriate action to address such determination.  The Director may require the injector to obtain an individual permit, require the injector to close the injection well, or take appropriate enforcement action including site remediation.

2.) This comment is specific to the [arb-water.pdf](https://github.com/uic-utah/uic-inventory/files/13746870/arb-water.pdf) template. On the first page, under the “APPROVAL AND AUTHORIZATION” section and after the first paragraph starting with “The Division of Water Quality (DWQ) has reviewed the…”:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/4787cbd9-6cf6-42db-9051-53974c31347e)

could we add the following paragraph:

Approval is hereby granted to construct and operate NUMBER (#) Class V well(s) under Authorization by Rule according to the Administrative Rules for the Utah UIC Program (Utah UIC Program Rules), R317-7.

3.) This comment is also specific to the [arb-water.pdf](https://github.com/uic-utah/uic-inventory/files/13746870/arb-water.pdf) template. On the first page under the “APPROVAL AND AUTHORIZATION” section, it looks like the return command didn’t execute between the 3rd and 4th paragraph, for example:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/2f5e0044-f345-4c4b-b049-fe666ae7ea79)

Other than those few minor things, the ABR letters are looking great!

**Comment by steveoh** [comment]
These have been pushed to dev. Please validate.

**Comment by nathankota** [comment]
## 1) Stormwater ABR ([arb-water.pdf](https://github.com/uic-utah/uic-inventory/files/13746870/arb-water.pdf))- Approval and Authorization section
Where it says:
![image](https://github.com/uic-utah/uic-inventory/assets/15659642/c32d79ee-bdaf-448b-9eac-722ed8685151)

Please change to 
> Approval is hereby granted to construct and operate "NUMBER (#)" Class V well(s) under Authorization by Rule...

Where "NUMBER (#)" is the mathematical sum of well count, and spell out the number with the number notation in parentheses in the ABR. For example, based on the counts in the images below the ABR would read,
> _Approval is hereby granted to construct and operate four (4) Class V well(s) under Authorization by Rule..._
 
![image](https://github.com/uic-utah/uic-inventory/assets/15659642/a7650a05-ba28-43c6-907e-8c3c428aa060)
![image](https://github.com/uic-utah/uic-inventory/assets/15659642/f1e0b10e-3d79-47c1-80de-edbe6b86bc84)

### 2) All ABRs - Approval and Authorization section
Change the highlighted text in the following paragraph, from:
![image](https://github.com/uic-utah/uic-inventory/assets/15659642/ef54589d-e97b-498e-ad45-fce8e7829a69)
To:
_Please refer to the Well Information section of the included Inventory Information Report for any wells that are within an aquifer recharge discharge area or a source water protection zone. Wells in an aquifer recharge discharge area will be indicated by green “ARDA” text. Wells in a source water protection zone will be indicated by green “GWZ” text, and the associated Water System contact person will be indicated in the Water System Information section._

**Comment by rsparker-utah** [comment]
For my testing I created a new site (i.e., Site # 120), then created a well inventory for each of the five ABR letter types (i.e., General, LUWDS, SER, Vet, and Stormwater), approved each of these inventories, and then downloaded the completed report (i.e., ABR letter/Inventory Report) for each inventory to review. I have a few comments from my review:

1.) The SER (e.g., ABR from [site 120 inventory 216](https://uic-inventory.dev.utah.gov/review/site/120/inventory/216)) and Vet (e.g., ABR from [site 120 inventory 219](https://uic-inventory.dev.utah.gov/review/site/120/inventory/219)) ABR letters that are generated from the web application will need a similar adjustment to the Stormwater ABR that Nathan mentioned above. Specifically, on the first page of the aforementioned ABRs, in the second paragraph of the "APPROVAL AND AUTHROIZATION" section, could the "undefined" text:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/61c2fcad-79e4-4ffe-8758-d81aa18a3a16)
Please be changed to the format "NUMBER (#)", where "NUMBER (#)" is the mathematical sum of well count with the resulting number spelled out and the number notation in parentheses. For example, based on the counts in the following images:
![295736099-a7650a05-ba28-43c6-907e-8c3c428aa060](https://github.com/uic-utah/uic-inventory/assets/49735267/1c41590d-e9a8-4cac-a4aa-0d7e2d8b88cb)
![295736157-f1e0b10e-3d79-47c1-80de-edbe6b86bc84](https://github.com/uic-utah/uic-inventory/assets/49735267/b6662078-31eb-4342-8871-389ff50f6b81)

this section of the ABR would read:

_Approval is hereby granted to construct and operate four (4) Class V well(s) under Authorization by Rule according to the Administrative Rules for the Utah UIC Program (Utah UIC Program Rules), [R317-7](http://rules.utah.gov/publicat/code/r317/r317-007.htm)._

2.) I don't recall if this was mentioned specifically in another issue, so here is a bit of background information about the general ABR letter. The general ABR letter is meant to be a "catch all" for any submitted inventories that fall outside one of the other four main/common inventories (i.e., stormwater, SER, LUWDS, or vet). When a general inventory is submitted, the admin reviewer will adjust the inventory to the appropriate inventory class (based on the information provided in the submission) from the drop down menu on the Inventory Review Page, for example:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/ccddddc5-a11a-454b-968b-223439e26275)
Currently, when the admin reviewer makes this adjustment on the Inventory Review page, approves the inventory, and then downloads the completed packet, the updated inventory class on the subject  is not being populated on the ABR letter page. For example:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/bf55c175-3307-4a89-9571-da8213326042)
Could the inventory class be adjusted in the ABR to represent the updated value? For example, in [site 120 inventory 221](https://uic-inventory.dev.utah.gov/review/site/120/inventory/221), the inventory class was change from "General" to "Spent-Brine Return Flow" on the Inventory Review page and the resulting ABR letter would read:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/2ed29939-32bf-4321-ad3e-a58d605e49a7)
with the EPA well code value being referenced from [this table](https://docs.google.com/spreadsheets/d/1d59MZYqrfqzPSWIooQGXZUDRaPs1Lq1UiZ1rQpV2nAw/edit?usp=sharing).

3.) Also on the general ABR letter, after the first paragraph in the "APPROVAL AND AUTHORIZATION" section:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/b61780ee-71f9-43ba-8fc7-4ff9a0d6c2c3)

could we add the following two paragraphs: 

_Approval is hereby granted to construct and operate NUMBER (#) Class V well(s) under Authorization by Rule according to the Administrative Rules for the Utah UIC Program (Utah UIC Program Rules), [R317-7](http://rules.utah.gov/publicat/code/r317/r317-007.htm)._

_The Class V well(s) associated with this authorization are authorized as (WellSubclass). The subject site is authorized to dispose of wastewater in accordance with the activities defined in the submitted well inventory application associated with this Authorization by Rule approval._

where "NUMBER (#)" is the mathematical sum of well count with the resulting number spelled out and the number notation in parentheses, and (WellSubclass) is equal to the the Inventory Class selected on the Inventory Review Page:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/a10c5829-5452-4ef9-be0e-54a7c4a1dadf)

**Comment by rsparker-utah** [comment]
I was able to go through another round of testing and all the updates look great! I did come across a couple additional things during this round of testing that I missed during my first round of review. For this round of testing I again created a new site (i.e., Site # 121), then created a well inventory for each of the five ABR letter types (i.e., General, LUWDS, SER, Vet, and Stormwater), approved each of these inventories, and then downloaded the completed report (i.e., ABR letter/Inventory Report) for each inventory to review. My comments from this round of testing are as follows:

**1.) All ABR Letters**

Currently, all of the ABR letters display the contact's name, organization and city/state/zip at the top of the letter, for example:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/6046c103-129f-4a7a-858b-53787cc01566)
Could the address for the contact be included at the top of the letters along with the contact's name, organization and city/state/zip? For example:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/8c8b203b-40dc-42c3-af35-1d5731443005)


**2.) SER ABR Letter**
I missed a couple of typos I had made in the [original template for the SER ABR letter](https://docs.google.com/document/d/1-jptVL4pIhDmcWN26Z4TdusHNR_tVS5ntjIz9qoFHLo/edit?usp=sharing) (these have since been fixed in the template). Because of this, could we change the text under the "Approval and Authorization" section from:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/28981cd7-8e3b-498d-a597-329693b4947b)
to:
The construction and operation of any injection wells at this site for subsurface environmental remediation may be subject to additional requirements established by the Utah Division of Environmental Response and Remediation (DERR) or the Utah Division of Waste Management and Radiation Control (DWMRC).
